ledger/state.go

```diff
@@ -917,3 +942,68 @@ func (ls *LedgerState) ValidateTx(
 	}
 	return nil
 }
+
+// forgeBlock creates an empty conway block and logs its CBOR representation
+func (ls *LedgerState) forgeBlock() {
+	// Get current chain tip
+	currentTip := ls.Tip()
+
+	// Calculate next slot and block number
+	nextSlot := currentTip.Point.Slot + 1
```

Comment:
The slot number should be based on the current time instead of the previous block. You can use `ls.TimeToSlot(time.Now())` to calculate this.